addons/hr_recruitment/res_config.py

```diff
@@ -52,3 +52,17 @@ def set_default_alias_prefix(self, cr, uid, ids, context=None):
             else:
                 mail_alias.write(cr, uid, alias_id, {'alias_name': record.alias_prefix}, context=context)
         return True
+
+    def get_default_alias_domain(self, cr, uid, ids, context=None):
```

Comment:
This is fixed in this issue : https://www.odoo.com/web#id=643799&view_type=form&model=project.issue&menu_id=4120&action=609

And I don't think that this fix is correct. It's fixing only the issue in the hr_recruitment module, not in a generic way